.github/workflows/main.yml

```diff
@@ -62,10 +66,18 @@ jobs:
             toolchain: stable
             components: cargo
             image: ubuntu:groovy
+          - task: make -f Makefile release-min
+            toolchain: stable
+            components: cargo
+            image: ubuntu:groovy
           - task: make -f Makefile build
             toolchain: stable
             components: cargo
             image: fedora:32
+          - task: make -f Makefile build-min
```

Comment:
I think we should consider dropping the ```build-no-default``` Makefile task and the GitHub Actions directed at it. It was put in from a time where stratisd was never released without the defaults and wasn't really interesting without them. It was just useful for catching the occasional error when something didn't get the ```[cfg(dbus)]``` flag put on that should or vice-versa. But at this point, now that there are actual targets which build w/out dbus enabled, I think its usefulness is gone.